_episodes/05-access-data.md

```diff
@@ -442,3 +443,53 @@ b01.rio.to_raster("B01.tif")
 > ~~~
 > {: .language-python}
 {: .callout}
+
+> ## Exercise: Downloading data from a protected bucket
+>
+> Use `earthaccess` to download 10 files from the ASTGTM version 003 dataset.
```

Comment:
I think this exercises should also specify the collection id up front since learners won't know beforehand how to find that info to complete the search.